clearwater_cassandra/cassandra_plugin.py

```diff
@@ -246,6 +245,25 @@ def wait_for_cassandra(self):
         # scripts get run
         run_command(["sudo", "service", "clearwater-infrastructure", "restart"])
 
+    def add_cassandra_schemas(self):  # pragma: no cover
+        # Adding the schemas can fail when Cassandra has just started. If it
+        # does fail, simply try again until it succeeds.
```

Comment:
I haven't gone for a limit we can't use Cassandra at all if the schemas aren't loaded. 
Breaking means that we'd end up saying that we'd successfully clustered, the Cassandra alarms would clear, and it would be harder to debug what the problem is.